src/content/reference/manifest.mdx

```diff
@@ -72,6 +72,8 @@ You can build all these images by running `okteto build`, or `okteto build xxx`
 
 > if a `<service>` name includes the symbol `-`, in the corresponding `OKTETO_BUILD_<service>` environment variables, the symbol `-` will be replaced by `_`.  For example, if `<service>` is `name-with-hyphen`, the environment variable names will start with `$OKTETO_BUILD_NAME_WITH_HYPHEN_`.
 
+> Okteto autoinjects `OKTETO_NAMESPACE` and `OKTETO_CONTEXT` on every build as build args.
```

Comment:
"Okteto will automatically add OKTETO_NAMESPACE and OKTETO_CONTEXT on every build as a build argument."